app/src/main/kotlin/at/bitfire/davdroid/ui/AccountsScreen.kt

```diff
@@ -293,6 +306,34 @@ fun AccountsScreen(
                                     val intent = Intent(Settings.ACTION_INTERNAL_STORAGE_SETTINGS)
                                     if (intent.resolveActivity(context.packageManager) != null)
                                         context.startActivity(intent)
+                                },
+                                calendarStorageEnabled = calendarStorageEnabled,
+                                onManageCalendarStorage = {
+                                    if (calendarStorageEnabled == null)
+                                        uriHandler.openUri(manualIntroductionUri.toString())
+                                    if (calendarStorageEnabled == false) {
+                                        val intent = Intent(
+                                            /* action = */ Settings.ACTION_APPLICATION_DETAILS_SETTINGS,
+                                            /* uri = */
+                                            Uri.parse("package:com.android.providers.calendar")
+                                        )
+                                        if (intent.resolveActivity(context.packageManager) != null)
+                                            context.startActivity(intent)
```

Comment:
It's used twice – can we put it into a small method? Then we also don't need to define `manualIntroductionUri` for the whole scope.